clientd/Cargo.toml

```diff
@@ -12,6 +12,7 @@ futures = "0.3.25"
 jsonrpsee = { version = "0.16.1", features = [ "full" ] }
 serde = { version = "1.0.148", features = ["derive"] }
 serde_json = "1.0.89"
+structopt = "0.3.26"
```

Comment:
Didn't we replace all uses of `structopt` with `clap` in Fedimint since `clap` got all the functionality too now?

Reply:
yeah but structopt is smaller and there is no need for a real cli..(also you said todo: use structopt last PR ;)

Reply:
also I'm thinking of just introducing a configuration file for clientd later so cmd args will be very little if not none

Reply:
> structopt is smaller

structopt depends on clap …